packages/docs/src/components/page-configs/contribution/translation/page-config.ts

```diff
@@ -1,15 +1,18 @@
 import { ApiDocsBlock } from '@/types/configTypes'
 import { DocsHelper } from '@/helpers/DocsHelper'
-import { t } from '@/helpers/I18nHelper'
 import { languages } from '@/components/languages'
 
-const translationStatuses = languages.map((lang) => [`${lang.name} (${lang.code})`, lang.status])
+const translationStatuses = languages.map((lang) => [
+  `translation.language.${lang.code}`,
+  lang.code,
+  `translation.status.${lang.status}`
```

Comment:
Please use full translation string here. I.e. 'translation.status.full'.

Could be implemented as a map

```ts
const statusTranslations = {
  'full': 'translation.status.full',
  'partial': 'translation.status.partial'
}
```

Right now all strings in app are full. This allows to do automated check on which strings are being used.